content/docs/configuration/venafi.md

```diff
@@ -189,11 +184,19 @@ $ kubectl create secret generic \
 
 ### Username / Password Authentication
 
-NOTE: cert-manager uses the deprecated "API key Web SDK authentication" for
-authenticating with a username and password. Because cert-manager has not been
-updated to use the newer authentication, it will become necessary for you to
-request a license key to Venafi's customer support in order to continue using
-the username and password authentication with cert-manager.
+> ⚠️ When you supply a Venafi TPP username and password,
+> cert-manager uses an older authentication method which is called "API Keys",
+> which has been deprecated since Venafi TPP `19.2`.
+>
+> Beginning in Venafi TPP `22.2`, "API Keys" are disabled by default.
+> You will need to contact Venafi customer support for a special licence key which will allow you to re-enable the "API Keys" feature,
+> so that you can continue to use username and password authentication with cert-manager.
+>
+> In Venafi TPP `22.3`, the "API Keys" feature will be permanently removed,
+> and you will need to use access-token authentication instead.
```

Comment:
In https://github.com/cert-manager/website/pull/1094#discussion_r1000266914 @SpectralHiss wrote:
> Should note that in 22.3 API Key authentication is completely removed!
> But using user/pass doesn't mean necessarily using API Key no? unless that's how it's implemented in cert-manager


I've tried to address Houssem's comment in this modified version of the warning.